config/config.go

```diff
@@ -128,6 +146,7 @@ type LocalModeConfig struct {
 	ContainerRegistry     *ContainerRegistryConfig `yaml:"containerRegistry" json:"containerRegistry"`
 	RuntimeLimits         RuntimeLimits            `yaml:"runtimeLimits" json:"runtimeLimits"`
 	ForceEnableInspection bool                     `yaml:"forceEnableInspection" json:"forceEnableInspection"`
+	DeduplicationConfig   *DeduplicationConfig     `yaml:"deduplicationConfig" json:"deduplicationConfig"`
```

Comment:
```suggestion
	Deduplication   *DeduplicationConfig     `yaml:"deduplication" json:"deduplication"`
```
Can we avoid suffixing property names with "config"? We have been doing it like that and it keeps the names simpler.